workflows/n_t.snake

```diff
@@ -31,7 +30,7 @@ import masks
 # A seed to replicate results
 # TODO mutation rates
 
-configfile: "workflows/config/snakemake/config.yaml"
+configfile: "workflows/config/snakemake/production-config.yml"
```

Comment:
we don't want this checked in yet-- as we develop the workflow we want to be running config.yaml or the tiny config